README.md

```diff
@@ -44,22 +44,41 @@ All timestamps are captured with [performance.now()](https://developer.mozilla.o
 ### Before lauching an experiment
  Warning : tileserver use node v6 and selenium-webdriver use node v8
 
-* in data/ type ```nvm exec v6.14.1 tileserver-gl 2017-07-03_europe_switzerland.mbtiles ```
+* in data/ type ```docker run --rm -it -v $(pwd):/data -p 80klokantech/tileserver-gl 2017-07-03_europe_switzerland.mbtiles ```
```

Comment:
It looks like a part of `-p 8080:80` was cut. But anyway, this is no longer needed with now, is it?